### DISCUSSION TITLE
Apktool v2.6.0

### DISCUSSION THREAD

**Discussion by iBotPeaches** [iBotPeaches/Apktool#2646] [Releases]
https://connortumbleson.com/2021/09/02/apktool-v2-6-0-released/

<hr /><em>This discussion was created from the release <a href='https://github.com/iBotPeaches/Apktool/releases/tag/v2.6.0'>Apktool v2.6.0</a>.</em>

**Comment by ghost** [comment]
How do I start it?

**Reply by iBotPeaches** [reply]
`java -jar apktool.jar`

**Reply by adamjon23** [reply]
Thank you for sharing the link :)

**Comment by davidbhai1212** [comment]
how to i install /start it in kalilinux

**Reply by iBotPeaches** [reply]
https://ibotpeaches.github.io/Apktool/install/

**Comment by c3eru** [comment]
Sir,  request for support amaze apktool for a11-a12 
Thanks🙏

**Reply by iBotPeaches** [reply]
I don't know what this means.
